packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -27,6 +28,10 @@ export interface AudienceViewProps {
      */
     audience: IServiceAudience<IMember>;
 
+    /**
+     * Audience
```

Comment:
Can we expand this to explain a bit more about what it is?

Reply:
Fixed.